ssr.sh

```diff
@@ -759,9 +759,9 @@ Install_Libsodium(){
 		echo -e "${Info} 安装依赖..."
 		yum -y groupinstall "Development Tools"
 		echo -e "${Info} 下载..."
-		wget  --no-check-certificate -N "https://github.com/jedisct1/libsodium/releases/download/${Libsodiumr_ver}/libsodium-${Libsodiumr_ver}.tar.gz"
+		wget  --no-check-certificate -N "https://github.com/jedisct1/libsodium/archive/${Libsodiumr_ver}-RELEASE.tar.gz"
 		echo -e "${Info} 解压..."
-		tar -xzf libsodium-${Libsodiumr_ver}.tar.gz && cd libsodium-${Libsodiumr_ver}
+		tar -xzf ${Libsodiumr_ver}-RELEASE.tar.gz && cd ${Libsodiumr_ver}-RELEASE
```

Comment:
`cd ${Libsodiumr_ver}-RELEASE` should be `cd libsodium-${Libsodiumr_ver}-RELEASE`